test/inductor/test_mps_basic.py

```diff
@@ -211,7 +211,7 @@ def forward(self, x, y):
         path = torch._inductor.aoti_compile_and_package(ep, "here.pt2")
         m = torch._inductor.aoti_load_package(path)
         res = m(*inp)
-        assert torch.allclose(res, res2)
+        self.assertTrue(torch.allclose(res, res2))
```

Comment:
This will call https://github.com/pytorch/pytorch/blob/3f29642ecf039129032cc61c986d7b62807163c0/torch/testing/_internal/common_utils.py#L4010 which already accounts for default tolerances specific to dtypes....

Also, it feels like a BE change that deserves it's own small PR in the stack
```suggestion
        self.assertEqual(res, res2)
```